lib/ota/common.js

```diff
@@ -458,6 +458,9 @@ function getAxios() {
         config = {
             proxy: false,
             httpsAgent: new HttpsProxyAgent(proxy),
+            headers: {
+                'Accept-Encoding': '*',
```

Comment:
This says that **every** encoding is supported, which may not actually be true. It's better to listed the supported encodings along with quality values (e.g. to prefer Brotli over deflate and gzip), for example `'Accept-Encoding': 'br, gzip;q=0.9, deflate;q=0.9`